tools/istio-iptables/pkg/constants/constants.go

```diff
@@ -88,6 +88,7 @@ const (
 	CleanupOnly               = "cleanup-only"
 	ForceApply                = "force-apply"
 	NativeNftables            = "native-nftables"
+	ForceIptablesBinary       = "force-iptables-version"
```

Comment:
```suggestion
	ForceIptablesBinary       = "force-iptables-binary"
```

Reply:
fixed in https://github.com/istio/istio/pull/57976/commits/a97ba6839d1c1c2a16aa4c0cd085f6854fdfc244